### DISCUSSION TITLE
"Require holding key to pause gameplay" doesn't behave correctly globally/in multiplayer

### DISCUSSION THREAD

**Discussion by chiascuro** [ppy/osu#31033] [Q&A]
### Type

Game behaviour

### Bug description

I have the feature turned off, aka pressing ESC opens the pause menu instantly. Meanwhile in multiplayer, I have to hold the key to pause the game, despite it being disabled.

### Version

2024.1208.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18059568/compressed-logs.zip)

**Comment by frenzibyte** [comment]
You don't pause in multiplayer, you exit the match altogether. It's a dangerous operation and as such requires holding before it fires.
